source/community/about/community-guidelines.html.md

```diff
@@ -73,7 +73,7 @@ In general, we try to follow the [Wikipedia Style Guide](http://en.wikipedia.org
 
 ## Bug reporting guidelines
 
-*   Before reporting your bug, please [search the bug tracker](https://bugzilla.redhat.com/query.cgi?product=oVirt) to see if it has been reported already. A comment added to an existing bug report is also very useful.
+*   Before reporting your bug, please [search the bug tracker](https://bugzilla.redhat.com/query.cgi?classificiation=oVirt) to see if it has been reported already. A comment added to an existing bug report is also very useful.
```

Comment:
Close, but should probably be:
https://bugzilla.redhat.com/buglist.cgi?bug_status=__open__&classification=oVirt